Configure.cmake

```diff
@@ -9,6 +9,23 @@ find_library(LIBGMP gmp)
 find_library(LIBRT rt)
 find_library(LIBFFTW3 fftw3)
 
+if (NOT CMAKE_CROSSCOMPILING AND NOT SLEEF_FORCE_FIND_PACKAGE_SSL)
+  find_package(OpenSSL)
+else()
+  # find_package cannot find OpenSSL when cross-compiling
+  find_library(LIBSSL ssl)
+  find_library(LIBCRYPTO crypto)
+  if (LIBSSL AND LIBCRYPTO)
+    set(OPENSSL_FOUND TRUE)
+    set(OPENSSL_LIBRARIES ${LIBSSL} ${LIBCRYPTO})
```

Comment:
This is risky. The OPENSSL_* variables are set by "find_library" afai, and manually setting them might break other components if SLEEF is added as a cmake subproject. Please avoid doing so. If you want to be able to switch between openSSL and libssl, I recommend setting variables "SLEEF_MD5_FOUND" and "SLEEF_MD5_LIBRARIES" and "SLEEF_MD5_INCLUDES" according to the results of find_package on libopenssl and libssl, and then use those new variables in the rest of the CMAKE scripts.